src/components/Calculator.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
+  const [result, setResult] = useState('0');
+
+  const handleClick = (value) => {
+    setResult((prevResult) => {
+      if (prevResult === '0') {
+        return value;
+      }
+      return prevResult + value;
+    });
+  };
+
+  const handleCalculate = () => {
```

Comment:
- You have done great so far. :+1: :+1: But there are some requirements you still need to meet before moving to the next milestone. :+1: 
> Implement the design only (JSX and CSS)
Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- [ ] Following the requirement for this milestone, you are not meant to implement any functionality for the calculator app. You are meant to work only on the UI of the calculator. The code for the calculator logic will be provided in the upcoming Milestone. I'll suggest you stick to the requirement. :smile: :wave: